migrations/settings/0001-migrate-from-deprecated-icon-names.js

```diff
@@ -0,0 +1,748 @@
+export default function migrate(settings, helpers) {
+  const oldBannerLinksSetting = settings.get("banner_links") || "[]";
+  const newBannerLinksSetting = JSON.parse(oldBannerLinksSetting).map(
+    (link) => {
+      const newLink = {
+        icon: convertIconName(link.icon),
+        text: link.text.substring(0, 1000),
```

Comment:
No null checks on these since the previous json schema already has `minLength: 1` validations.